### ISSUE TITLE
Power mapper block

### ISSUE THREAD

**Issue by AmyJuneH** [badcamp/badcamp-2019#1]
https://app.powermapper.com/Scans/Report/368880

error message:
Some links were not visited because they were blocked by a ROBOTS.TXT file.
A ROBOTS.TXT file is a digital "keep out" sign placed on a web server by the server administrator. Adding the following entries to the bottom of the robots.txt file, after all other directives, will bypass any blocks intended for other web crawlers:
User-agent: PowerMapper
Allow: /

**Comment by AmyJuneH** [comment]
this is not a priority
